qos/evm/evm.go

```diff
@@ -56,6 +56,20 @@ func (qos *QoS) ParseHTTPRequest(_ context.Context, req *http.Request) (gateway.
 	}, true
 }
 
+// ParseWebsocketRequest builds a request context from the provided WebSocket request.
+// WebSocket connection requests do not have a body, so we don't need to parse it.
+//
+// This method implements the gateway.QoSService interface.
+// TODO_HACK(@commoddity, WebSockets): Utilize this method once the Shannon protocol supports websocket connections.
```

Comment:
@commoddity Can you create a ticket to productionize websockets and then replace all `TODO_HACK(@commoddity, WebSockets)` with `TODO_HACK(@commoddity, #XXX)`?

Reply:
Feel free to use a screenshot of this thread for reference + link to #135

Reply:
Done.